airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/CatalogItems.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "CatalogItems",
+  "description": "Catalog Items",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "numberOfResults": { "type": ["null", "number"] },
+    "items": {
+      "type": ["null", "array"],
+      "properties" : {
+        "asin":  {"type": ["null", "number"]},
+        "attributes":  {"type": ["null", "object"]},
```

Comment:
Please check https://docs.airbyte.com/connector-development/schema-reference add `additionalProperties` to object fields. Would be better to add the items for each array if possible.